apps/website/.vuepress/theme/global-components/DocCode.vue

```diff
@@ -53,6 +53,7 @@ export default {
 .code-wrapper {
   position: relative;
   padding: 0.6rem;
+  margin-top: 0.25rem;
   border: 1px solid #ccc;
```

Comment:
Do we not want to use rem values and/or tokens for the border here?

Reply:
That is existing code outside of this PR, but there are a few places we should update to address these kind of issues.